starknet_py/net/models/transaction.py

```diff
@@ -105,6 +120,20 @@ def calculate_hash(self, chain_id: StarknetChainId) -> int:
             nonce=self.nonce,
         )
 
+    @staticmethod
+    def _contract_attribute_name(data: Dict[str, Any]):
+        return (
+            "contract_definition" if "contract_definition" in data else "contract_class"
```

Comment:
I think it's backward compatibility, no?

Reply:
It is. It's not expensive to have it though, so I think it's fine.